libconnman-qt/networkmanager.cpp

```diff
@@ -86,8 +88,9 @@ class NetworkManager::Private : public QObject
         , m_connected(false)
         , m_connectedWifi(NULL)
         , m_connectedEthernet(NULL) {}
+
     NetworkManager* manager()
-        { return (NetworkManager*)parent(); }
+        { return (NetworkManager*) parent(); }
```

Comment:
nit: c-style cast caught my eye... would qobject/dynamic/static_cast be more appropriate?